internal/support/scheduler/router.go

```diff
@@ -120,12 +115,7 @@ func loadRestRoutes(r *mux.Router, dic *di.Container) {
 	interval.HandleFunc(
 		"/"+NAME+"/{"+NAME+"}",
 		func(w http.ResponseWriter, r *http.Request) {
-			restDeleteIntervalByName(
-				w,
-				r,
-				bootstrapContainer.LoggingClientFrom(dic.Get),
-				container.DBClientFrom(dic.Get),
-				schedulerContainer.QueueFrom(dic.Get))
+			restDeleteIntervalByName(w, r, bootstrapContainer.LoggingClientFrom(dic.Get), container.DBClientFrom(dic.Get), schedulerContainer.QueueFrom(dic.Get), container.DBClientFrom(dic.Get), container.DBClientFrom(dic.Get))
```

Comment:
Can we put these arguments on new lines like it was previously?

Reply:
Can we look into applying this previous comment?

Reply:
Done (Addressed both reformatting-related comments) ✅